src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -1294,9 +1313,30 @@ bool RekordboxPlaylistModel::isColumnHiddenByDefault(int column) {
 
 bool RekordboxPlaylistModel::isColumnInternal(int column) {
     return column == fieldIndex(ColumnCache::COLUMN_REKORDBOX_ANALYZE_PATH) ||
+            column == fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_PLAYED) ||
             BaseExternalPlaylistModel::isColumnInternal(column);
 }
 
+void RekordboxPlaylistModel::onPlayingTrackChanged(TrackPointer pTrack) {
+    if (pTrack) {
```

Comment:
prefer early returns
```suggestion
    if (!pTrack) {
        return;
    }
```